test/unit/lib/env.test.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef */
+describe('env', () => {
+
+  describe('load default values without override', () => {
+
+    const envTest = require('../../../lib/env')
+
+    it('loads default CONFIG_PATH if not passed', () => {
+      const CONFIG_PATH = envTest.CONFIG_PATH
+      expect(CONFIG_PATH).toEqual('.github')
+    })
+
+    it('loads default SETTINGS_FILE_PATH if not passed', () => {
+      const SETTINGS_FILE_PATH = envTest.SETTINGS_FILE_PATH
+      expect(SETTINGS_FILE_PATH).toEqual('settings.yml')
+    })
+
+    it('loads default ADMIN_ORG if not passed', () => {
```

Comment:
Does it make sense to allow the "admin" repo be in a different organization? Maybe we should use here `GH_ORG` instead?

Reply:
What is `GH_ORG`? is that an already defined env variable? what do you mean with "to allow the "admin" repo be in a different organization", this change does not cover that only the customizable org/repo base. Is that a question for an enhancement? Thanks!

Reply:
I might misunderstand something but from what I know safe-settings works like that:
* create a GH app within `orgA`
* configure safe-settings probot app and set GH_ORG, APP_ID, WEBHOOK_SECRET, etc.

By doing that safe-settings will listen on events on GH_ORG (where the app was installed). If now you decide to set `ADMIN_ORG` on a different organization than `orgA` I guess certain things won't work, like getting events for repo `ADMIN_ORG:admin` from the GitHub App.

So to me it seems `ADMIN_ORG` should point always to `GH_ORG` or do I misunderstand something here?

Reply:
That GH_ORG is a manifest variable to set the app registered to an organization (like kind of owner/developer) but it doesn't need to be set, as if not it gets sets to the user that requests it.

https://probot.github.io/docs/configuration/

```
The organization where you want to register the app in the app creation manifest flow. 
If set, the app is registered for an organization (https://github.com/organizations/ORGANIZATION/settings/apps/new), if not set, the GitHub app would be registered for the user account (https://github.com/settings/apps/new).
```

I think that is different than the env variable we are trying to add here, that is behavioural. Here we are trying to say: Instead of looking for an org or repo called `admin` we allow GitHub App owners (the GH_ORG or the user owner) to say that this instance of the app should look for a different org or repo as admin)

As an example I would say that several probot apps use `.github` repo as default kind of admin, were a profile file can be added so it is shown as a introduction when the user is in the org page. Based on that a bunch of apps are using that `.github` repo as admin/default, but safe-settings is looking for `admin` repo for such purpose. This change allows to set `.github` repo as admin, so we can have all in one place.

Reply:
Sorry for the late response. 

To be clear: the authentication between the Probot app and the "admin" repo is achieved through the APP_ID and PRIVATE_KEY which is registered on GH_ORG, right?

If now GH_ORG != ADMIN_ORG, how will the Probot app be able to authenticate with the ADMIN_ORG (especially in an organization with private repos)?

Just that it is clear: I really love this change as one can finally use a different repository instead admin and especially the CONFIG_PATH which would allow us to move out the configuration to a different folder. I definitely will use this new feature once it is merged. I just do not understand yet the need for a new ADMIN_ORG variable as I believe it would be better to not allow different values here.

Reply:
Removed ADMIN_ORG change